src/common/modules/Logger.js

```diff
@@ -13,6 +13,36 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Checks if value is a object.
+ *
+ * @function
+ * @param  {*} value
+ * @returns {boolean}
+ */
+function isObject(value){
+    const type = typeof value;
+    return value != null && (type === "object" || type === "function");
+}
+
+/**
+ * Copy nested objects to ensure a properly output.
+ *
+ * @function
+ * @param  {...*} args
+ * @returns {Object}
+ */
+function verifyArgs(args) {
+    for(const key in args) {
+        if (args.hasOwnProperty(key)) {
+            if(isObject(args[key])) {
```

Comment:
Well why do you nest that? You could just use "and" (`&&`) to link them. Generally anyway, when you are having a look here: The `Contributing.md` states you should use early return, i.e. check for errors and continue.
Okay, I admit it has to be called "early continue" or so here, but yeah, you may anyway adjust it.

In any case, do whatever you want, but this does not need to be nested.